client/llb/state.go

```diff
@@ -269,6 +269,24 @@ func WithDescription(m map[string]string) MetadataOpt {
 	})
 }
 
+func WithExportCache() MetadataOpt {
+	return metadataOptFunc(func(md *OpMetadata) {
+		md.ExportCache = &pb.ExportCache{Value: true}
+	})
+}
+
+func WithoutExportCache() MetadataOpt {
+	return metadataOptFunc(func(md *OpMetadata) {
+		md.ExportCache = &pb.ExportCache{Value: false}
+	})
+}
+
+func WithoutDefaultExportCache() MetadataOpt {
```

Comment:
Could you add godoc about the difference between `md.ExportCache = nil` and `md.ExportCache=&{...false}`? (and also to ops.proto)